cli/src/main/java/org/infinispan/cli/commands/rest/Shutdown.java

```diff
@@ -78,7 +78,7 @@ protected CompletionStage<RestResponse> exec(ContextAwareCommandInvocation invoc
       }
    }
 
-   @CommandDefinition(name = "container", description = "Shuts down the container without terminating the server processes", activator = ConnectionActivator.class)
+   @CommandDefinition(name = "container", description = "Stops the data container without terminating the server process. Intended for use with the Kubernetes Operator only.", activator = ConnectionActivator.class)
```

Comment:
"Intended for use with the Kubernetes Operator only".

That's too far, as the Operator doesn't even use the CLI command. There might be some other k8 workloads where this CLI command makes sense. I would just remove this last sentence.

Reply:
Thanks, @ryanemerson I've removed that and reworded the manpage. I think Wolf wanted to have the same hint in both places but it's too much info to fit into the `-h` output.